src/editor.ts

```diff
@@ -0,0 +1,15 @@
+import { ImageEditorWidget } from './widget';
+import { DocumentRegistry, DocumentWidget } from '@jupyterlab/docregistry';
+
+export namespace ImageEditorDocumentWidget {
+    export interface IOptions<T>
+      extends DocumentWidget.IOptions<ImageEditorWidget, DocumentRegistry.IModel> {
+    }
+  }
+
+export class ImageEditorDocumentWidget extends DocumentWidget<ImageEditorWidget> {
+
+  constructor(options: ImageEditorDocumentWidget.IOptions<ImageEditorWidget>) {
+    super(options)
+  }
+}
```

Comment:
This file can actually be removed because you don't need any customization of `DocumentWidget`